### PR TITLE
Domains: Use new component for domain notices

### PR DESCRIPTION
**Pull request by eltongo**
#### Changes proposed in this Pull Request

This PR adds a new component to use for domain notices, instead of the old `Notice` component.

Screenshot 2020-06-16 at 7 17 12 PM

#### Testing instructions

For a domain of yours that has expired or is near expiration, a notice similar to the attached screenshot above should be shown.

### CODE REVIEW

File: client/my-sites/domains/domain-management/list/item.jsx [hunk 1]
```diff
@@ -159,29 +159,30 @@ class ListItem extends React.PureComponent {
 
 		if ( domain.expired ) {
 			return (
-				<Notice isCompact status="is-error" icon="spam">
-					{ translate( 'Expired %(timeSinceExpiry)s', {
+				<DomainNotice
+					status="alert"
+					text={ translate( 'Expired %(timeSinceExpiry)s', {
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/list/item.jsx:164]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/domains/domain-management/list/item.jsx [hunk 2]
```diff
@@ -159,29 +159,30 @@ class ListItem extends React.PureComponent {
...
 			);
 		}
 
 		if ( domain.expiry && moment( domain.expiry ) < this.props.moment().add( 30, 'days' ) ) {
 			return (
-				<Notice isCompact status="is-error" icon="spam">
-					{ translate( 'Expires %(timeUntilExpiry)s', {
+				<DomainNotice
+					status="warning"
+					text={ translate( 'Expires %(timeUntilExpiry)s', {
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/list/item.jsx:179]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Comment by fditrapani** [comment]
Nice! I see mine on a mapped domain:

image

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3914107

Thank you @eltongo for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3914107).

**Pull request merged**